app/assets/javascripts/analytics/static-analytics.js

```diff
@@ -207,6 +209,26 @@
     this.setDimension(21, position);
   };
 
+  StaticAnalytics.prototype.setTotalNumberOfSections = function() {
+    var sidebarSections = $('.govuk-related-items h2').length;
```

Comment:
Agreed these selectors know too much about the internals of the component, and will break without tests.

Not thought it through but maybe we can put some data attributes in the component?